modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -88,13 +92,45 @@ const setup = (editor: Editor): RenderInfo => {
     }
   });
 
-  const lazyHeader = () => lazyOuterContainer.get().bind(OuterContainer.getHeader);
-  const lazySinkResult = () => Result.fromOption(lazySink.get(), 'UI has not been rendered');
-  const lazyAnchorBar = () => lazyOuterContainer.get().bind((container) => memAnchorBar.getOpt(container)).getOrDie('Could not find a anchor bar element');
-  const lazyToolbar = () => lazyOuterContainer.get().bind((container) => OuterContainer.getToolbar(container)).getOrDie('Could not find more toolbar element');
-  const lazyThrobber = () => lazyOuterContainer.get().bind((container) => OuterContainer.getThrobber(container)).getOrDie('Could not find throbber element');
-
-  const backstage: Backstage.UiFactoryBackstage = Backstage.init(lazySinkResult, editor, lazyAnchorBar);
+  const lazyHeader = () => lazyUiRefs.mainUi.get()
+    .map((ui) => ui.outerContainer)
+    .bind(OuterContainer.getHeader);
+
+  const lazyDialogSinkResult = () => Result.fromOption(
+    lazyUiRefs.dialogUi.get().map((ui) => ui.sink),
+    'UI has not been rendered'
+  );
+
+  const lazyPopupSinkResult = () => Result.fromOption(
+    lazyUiRefs.popupUi.get().map((ui) => ui.sink),
+    '(adjacent) UI has not been rendered'
```

Comment:
Is this still correct as it's using a different name? Should it be this instead:
```suggestion
    '(popup) UI has not been rendered'
```

Reply:
Yep. I changed the names between the first spike and this task. It should be popup for now

- [ ] change all usages of adjacent to popup